README.md

```diff
@@ -529,6 +529,13 @@ bind mount (i.e., a filesystem path volume). To switch back to a pre-built versi
 This will work for any service where both a local development and pre-built image exist.
 
 
+### Using Adminer
+
+An [Adminer](https://www.adminer.org/) container is deployed in dev and local mode, it can be used to inspect the Katsu database during development. 
+
+Go to `localhost:8080` to access the login page, fill the fields with the values shown bellow, use the value of `BENTOV2_KATSU_DB_PASSWORD` in the password field.
```

Comment:
wrap lines to 120 - also, "bellow" should be "below" instead, and this is a run-on sentence, so it should be more like

Go to `localhost:8080` to access the login page. Fill the fields with the values shown below, using the value of `BENTOV2_KATSU_DB_PASSWORD` for the password field.